sdk/containerregistry/Azure.Containers.ContainerRegistry/CHANGELOG.md

```diff
@@ -1,14 +1,6 @@
 # Release History
 
-## 1.1.0-beta.10 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.1.0 (2023-05-09)
```

Comment:
I believe some text is required in the changelog. We could put something like

Features Generally Available from 1.1.0-beta.9

Reply:
Cool, yeah, I was going to update it once I addressed https://github.com/Azure/azure-sdk-for-net/issues/35771
Happy to add this additional text - thanks!

Reply:
Added.